clusters/hive/pools/cvp/cvp-ocp-4-6-amd64-aws-us-east-1_clusterpool.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: hive.openshift.io/v1
+kind: ClusterPool
+metadata:
+  name: cvp-ocp-4-6-amd64-aws-us-east-1
+  namespace: cvp-cluster-pool
+  labels:
+    product: ocp
+    version: "4.6"
+    architecture: amd64
+    cloud: aws
+    owner: cvp
+    region: us-east-1
+spec:
+  baseDomain: cpaas-ci.devcluster.openshift.com
+  imageSetRef:
+    name: ocp-4.6.0-amd64
+  installConfigSecretTemplateRef:
+    name: install-config-aws-us-east-1
+  skipMachinePools: true
+  platform:
+    aws:
+      credentialsSecretRef:
+        name: cvp-aws-credentials
+      region: us-east-1
```

Comment:
I believe we will need to use a different region for each cluster pool. I propose the us-west-1 region for v4.6.

Reply:
ack